src/org/intellij/grammar/LightPsi.java

```diff
@@ -124,11 +124,12 @@ private static boolean shouldAddEntry(String path) {
     if (path.contains("/grammar-kit/")) return false;
 
     return path.contains("/out/classes/production/") ||
-           path.contains("idea.jar") ||
-           path.contains("platform-api.jar") ||
-           path.contains("platform-impl.jar") ||
            path.contains("util.jar") ||
-           path.contains("extensions.jar");
+           path.contains("util-8.jar") ||
```

Comment:
why this change?

Reply:
To reflect changes in the manifest file. Some libraries mentioned in this code no longer exist, and some required libraries were missing. So I rewrote that since I needed to be able to run grammar-kit code from a .jar file.

Reply:
The JAR file names depend on the IDE version/distribution. I would expect it to no longer work with older IDE versions if removing entries.

Reply:
I assumed that JAR names depend on which SDK is used during build. I will re-add removed entries to the condition.